packages/key-did-resolver/src/nist_weierstrass_common.ts

```diff
@@ -0,0 +1,183 @@
+// Brent Shambaugh <brent.shambaugh@gmail.com>. 2021.
+
+import * as u8a from 'uint8arrays'
+import { base64url } from 'multiformats/bases/base64'
```

Comment:
We should probably merge and incorporate these changes first: 
https://github.com/ceramicnetwork/js-ceramic/pull/2279

Reply:
Will simplify our code a bunch.

Reply:
yeah that makes sense, should be merged/released in ceramic first and then can be moved over here